docs/api/document.rst

```diff
@@ -518,6 +518,13 @@ This class represents a document. It can be constructed from a file or from memo
 
       .. note:: In a typical use case, a page :ref:`Pixmap` should be taken after annotations / widgets have been added or changed. To force all those changes being reflected in the page structure, this method re-instates a fresh copy while keeping the object hierarchy "document -> page -> annotations/widgets" intact.
 
+   .. method:: Recolor(int pageNum, int colorNum)
```

Comment:
This method should be defined in the list at the top of the page, e.g. line 95: `:meth:`Document.ReloadColor`          ...